backend/src/routes/set.ts

```diff
@@ -0,0 +1,10 @@
+import { sql } from "@/lib/db";
+import { Elysia } from "elysia";
+
+const setRoute = new Elysia({ prefix : "/set"})
+  .post("/change-session-expired-date", 
+
+  sql`
+    
+  `
+  )
```

Comment:
What is this?

Reply:
Why is it separate from auth route then. You should name it something like `/sign-out` not `/change-session-expired-date`, Just remove this before merge and then create new issue like you mentioned

Reply:
understood